README.rst

```diff
@@ -66,16 +66,24 @@ Command line usage:
     wlc download
     wlc upload
 
-Configuration is stored in ``~/.config/weblate``:
+Configuration is stored in ``~/.config/weblate``. The key/values (``retries``,
```

Comment:
Is it possible to set those via environment variables? In docker environments it's sometimes tricky to inject a file?

Reply:
The existing pattern for configuration is to parse them directly as a parameter to the `wlc.Weblate` class or by adding them to a config file.

I am currently using env vars to configure our `wlc.Weblate` object in Dragoman by parsing them directly to the class upon initilisation:

```python
Weblate(
    key=os.environ.get('WEBLATE_KEY', None),
    url=os.environ.get('WEBLATE_URL', None),
    retries=os.environ.get('RETRIES', 3),
    status_forcelist=os.environ.get(
        'STATUS_FORCELIST',
       [429, 500, 502, 503, 504]
    )
    method_whitelist=os.environ.get(
        "METHOD_WHITELIST",
        ["HEAD", "TRACE", "DELETE", "PUT", "OPTIONS", "GET", "POST"]
    )
    backoff_factor=os.environ.get("BACKOFF_FACTOR", 0.2),
)
```
We can therefore parse the request retry configurations in dragoman-ui and dragoman apps `APP_SETTINGS_YAML` and then export them to the environment when running `app.py`. This is how I have been configuring it with Dragoman when testing my weblate integration changes.

I have allowed this to continue by adding parameters for the request configuration to the [`wlc.Weblate` class](https://github.com/superDross/wlc/blob/a7d931a119cc8da565e65d07bbb42db116b5c54b/wlc/__init__.py#L66).

I don't think we need to add the ability to configure `wlc` directly with env vars.

Reply:
Makes sense David - thanks for a detailed explanation.